### DISCUSSION TITLE
How do i change Jest extension icon

### DISCUSSION THREAD

**Discussion by liorf1** [orta/vscode-jest#1062] [Q&A]
I'm trying to make some adjustment to the extension and one of them is to change the jest icon on the left bar (the primary side bar). however this is not working by changing the icon path in the Package.json file. Anyone has any idea how to change that?
Thanks

**Comment by connectdotz** [comment]
If I understand your question correctly, you were trying to change the "beaker" icon? I have never tried it, but wondered if the [product-icon-theme](https://code.visualstudio.com/api/extension-guides/product-icon-theme) is the right place to look...
